nixos/tests/all-tests.nix

```diff
@@ -993,6 +993,7 @@ in {
   xterm = handleTest ./xterm.nix {};
   xxh = handleTest ./xxh.nix {};
   yabar = handleTest ./yabar.nix {};
+  your_spotify = handleTest ./your_spotify.nix {};
```

Comment:
nixos tests should be in its own commit.

Reply:
Should be fixed now.